### DISCUSSION TITLE
Threat Mapper not working,

### DISCUSSION THREAD

**Discussion by rmlawcadia** [deepfence/ThreatMapper#678] [General]
Hi, ive install threat mapper on newly created server,  And yes it was successfully intstall.  But when i go to the IP address its not working or booting the registration page. what should i do ?  Sorry im a newbie if you find this question stupid.

**Comment by ramanan-ravi** [comment]
Hi @rmohammad25, can you check if all containers are running?
```
docker ps -a
docker logs deepfence-api
```

**Reply by rmlawcadia** [reply]
curl: (6) Could not resolve host: deepfence-es Elastic search is unavailable - sleeping

![image](https://user-images.githubusercontent.com/74220570/202093366-30ca1406-bb80-457c-b933-4ecdd7e10e97.png)

**Comment by rmlawcadia** [comment]
Hi @ramanan-ravi, the reason why its not going to the initial configuration page is that 

the deepfence-es-master is always restating. 
![image](https://user-images.githubusercontent.com/74220570/202097685-760bf548-d289-4d17-96f2-2ebf30e3df0c.png)

its because of this error , when i do "docker logs -f deepfence-es-master"

![image](https://user-images.githubusercontent.com/74220570/202097899-c6fc13dc-1529-44e3-915a-510bd33b925a.png)

Should i install Java on the server ive created ? or what should i do?

**Reply by rmlawcadia** [reply]
@ramanan-ravi if you can give me your insights on this one. Please let me know

**Comment by shyam-dev** [comment]
Hello @rmohammad25  Please track this here - https://github.com/deepfence/ThreatMapper/issues/683 Thanks !

**Comment by shyam-dev** [comment]
Also @rmohammad25  - Did you see this response ? https://deepfence-community.slack.com/archives/C0156M8GV6Y/p1668577715807999?thread_ts=1668574189.537019&cid=C0156M8GV6Y
